src/api/spec/bootstrap/features/webui/users/profile_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'browser_helper'
+
+RSpec.feature 'User Profile', type: :feature, js: true do
```

Comment:
Why is this a feature and not a controller spec?

Reply:
I wanted to check the JS was running correctly and it really changed the `switch`after clicking. And that implies a change in database.

Reply:
To cover the beta program switch, not only what is on the controller.

Reply:
That is really costly just for checking a javascript oneliner with no logic that can break...

```
$('#beta-switch').on('click', function (){ $('#beta-form').submit(); });
```

Reply:
Until the day one of the IDs changes and we forget to update the JS. Shit happens.

Reply:
If you think this is likely and this is a good trade off, okay.

Reply:
I prefer to have this spec and not have to think about it by knowing it's covered.

Reply:
> I prefer to have this spec and not have to think about it by knowing it's covered.

I think the same. Maybe in some other cases it's too much (when a simple JS to add cool effect) but in this case, we are switching on/off a very important feature.